README.md

```diff
@@ -23,6 +23,32 @@ You have two options of how you want to build your UI
 - Using CRA default `yarn start` which will load your application as you would expect CRA to do. 
 - Using storybook by adding a `*.stories.tsx` and viewing your application in storybook allowing for multiple instances and setups along with individual components.
 
+### Deployment
+
+**Single Page Application (SPA)**
+
+Use the `yarn build` to produce a `build/{all application assets}` directory which contains the assets needed to publish. 
+
+**Shared UI**
+
+If you are building a shared UI you will need to follow the version strategy described below. Before each PR completion you need to update the version using the `yarn version:bump` script included in this template. For bundling use the `yarn build:ui` which does a check to see if the version exists on the CDN you are deploying to. 
```

Comment:
I wonder if for ease of use in the future if we could put a property in the package.json that you set to `spa` or `shared-ui` and then when you run `yarn build` it would just handle this for you, that way your not needing to remember this

Reply:
I like your idea but what is needed here isn't much different. Maybe I should add this to the docs here let me know; aside from the need of version bumping, you need to set this value in your build docker files, so it is set it and forget it. 

Locally you can still use the `yarn start` the `yarn build:ui` is just for moving the produced assets into a versioned folder.

Reply:
ah right you wouldn't have to do this while deving locally so not necessary then.

do we have it mentioned somewhere else in the docs that you have to change the build task in your dockerfile?

Probably doesn't hurt to put a link to that here